bridgenode/server.go

```diff
@@ -34,7 +37,7 @@ func blockServer(endHeight int32, dataDir string, haltRequest, haltAccept chan b
 			close(cons)
 			return
 		case con := <-cons:
-			go serveBlocksWorker(con, endHeight, dataDir)
+			go serveBlocksWorker(con, endHeight, dataDir, lvdb)
```

Comment:
Right, now the server needs access to the db to get the TTL values.  OK for now but we should fix this, as that could be a bottleneck for serving to lots of clients.
Probably should have the DB for only temporary use and write the TTL somewhere in the flat files once it's known.

Reply:
I think if the leveldb has enough cache performance shouldn't be too bad, but yeah having the ttls in flat files would be nicer.